src/App.tsx

```diff
@@ -1,52 +1,31 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
-
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
+import { Route, Routes } from 'react-router-dom';
+import { Navigation } from './components/Navigation';
+import { Tabs } from './components/Tabs';
 
 export const App = () => (
   <>
-    {/* Also requires <html class="has-navbar-fixed-top"> */}
-    <nav
-      className="navbar is-light is-fixed-top is-mobile has-shadow"
-      data-cy="Nav"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a href="/" className="navbar-item is-active">Home</a>
-          <a href="/tabs" className="navbar-item">Tabs</a>
-        </div>
-      </div>
-    </nav>
-
-    <div className="section">
-      <div className="container">
-        <h1 className="title">Home page</h1>
-        <h1 className="title">Tabs page</h1>
-        <h1 className="title">Page not found</h1>
-
-        <div className="tabs is-boxed">
-          <ul>
-            <li data-cy="Tab" className="is-active">
-              <a href="#/">Tab 1</a>
-            </li>
-            <li data-cy="Tab">
-              <a href="#/">Tab 2</a>
-            </li>
-            <li data-cy="Tab">
-              <a href="#/">Tab 3</a>
-            </li>
-          </ul>
-        </div>
+    <Navigation />
 
-        <div className="block" data-cy="TabContent">
-          Please select a tab
-        </div>
-      </div>
-    </div>
+    <Routes>
+      <Route path="/" element={<h1 className="title">Home page</h1>} />
+      <Route path="tabs">
+        <Route
+          index
+          element={(
+            <Tabs />
+          )}
+        />
+        <Route
+          path=":tabId/*"
```

Comment:
what is this `/*` for?

Reply:
there was some error pooping out in vsCode